### DISCUSSION TITLE
texify: security risk: running with elevated privileges

### DISCUSSION THREAD

**Discussion by HalilIbrahimAndic** [MiKTeX/miktex#1092] [Q&A]
Hi,

When I try to compile an article in "pdfLatex + MakeIndex + BibTeX" mode, this message appears: "texify: security risk: running with elevated privileges". 

If I choose "PdfLaTeX", this message appears but it compiles and give me pdf: "miktex-pdftex: security risk: running with elevated privileges"
[privilage_error.txt](https://github.com/MiKTeX/miktex/files/8505372/privilage_error.txt)

**Comment by HalilIbrahimAndic** [comment]
ReportDate: 2022-04-18 15:15:44
CurrentVersion: 22.3
SetupDate: 2022-04-18 15:05:00
SetupVersion: 22.3
Configuration: Regular
GitInfo: 5bfe728 / 2022-03-29 11:22:26
OS: Windows 10 Pro, 64-bit, build 19043
SharedSetup: no
LinkTargetDirectory: C:\Users\Halil Ibrahim Andic\AppData\Local\Programs\MiKTeX\miktex\bin\x64
PathOkay: yes
LastUpdateCheck: 2022-04-18 15:06:02
LastUpdate: 2022-04-18 15:06:30
LastUpdateDb: 2022-04-18 15:06:01
SystemAdmin: yes
RootPrivileges: yes
AdminMode: no
Root0: C:\Users\Halil Ibrahim Andic\AppData\Roaming\MiKTeX
Root1: C:\Users\Halil Ibrahim Andic\AppData\Local\MiKTeX
Root2: C:\Users\Halil Ibrahim Andic\AppData\Local\Programs\MiKTeX
UserInstall: C:\Users\Halil Ibrahim Andic\AppData\Local\Programs\MiKTeX
UserConfig: C:\Users\Halil Ibrahim Andic\AppData\Roaming\MiKTeX
UserData: C:\Users\Halil Ibrahim Andic\AppData\Local\MiKTeX

**Comment by HalilIbrahimAndic** [comment]
Texify Log: 

2022-04-18 15:08:28,613+0300 INFO  texify - this process (7260) started by miktex-texworks in directory D:\xxxx with command line: "C:\Users\Halil Ibrahim Andic\AppData\Local\Programs\MiKTeX\miktex\bin\x64\texify.exe" --pdf --synctex=1 --clean 2_Manuscript_982690.txt
2022-04-18 15:08:28,613+0300 WARN  texify - security risk: running with elevated privileges
2022-04-18 15:08:28,628+0300 INFO  texify.core - start process: miktex-tex.exe C:\Users\HALILI~1\AppData\Local\Temp\mik77361\txiversion.tex

**Comment by xiaoxstz** [comment]
I also came across this issue. The warning message is `bibtex: security risk: running with elevated privileges`

**Reply by HalilIbrahimAndic** [reply]
I am sorry to hear that. Still suffering from the issue but learn to use only PdfLatex.

**Comment by dragonhhhhh** [comment]
Have you solve the problem?
